Cargo.toml

```diff
@@ -42,7 +42,7 @@ proptest-attr-macro = "1.0.0"
 proptest-derive = "0.5.0"
 
 [lints.rust]
-unexpected_cfgs = { level = "deny", check-cfg = ['cfg(tarpaulin_include)'] }
+unexpected_cfgs = { level = "deny", check-cfg = ['cfg(feature, values("test"))', 'cfg(tarpaulin_include)'] }
```

Comment:
This change resolves the `error: unexpected 'cfg' condition value: 'test'` that has been observed on nightly for a while and came to stable with `1.84`.